app/queue/ingress.py

```diff
@@ -135,7 +135,8 @@ def handle_message(message, event_producer):
 
         event = build_event(event_type, output_host, metadata)
 
-        event_producer.write_event(event)
+        host_id = output_host["id"]
+        event_producer.write_event(event, host_id)
```

Comment:
Just making a bit more concise.

```suggestion
        event_producer.write_event(event, output_host["id"])
```